### DISCUSSION TITLE
[Request] 希望能增加一个话题置顶功能，可以选择需要重点关注的话题置顶

### DISCUSSION THREAD

**Discussion by ccdanpian** [lobehub/lobehub#1726] [General Question] [answered]
### 🥰 需求描述

![image](https://github.com/lobehub/lobe-chat/assets/130242490/35388374-d185-4d1e-89a5-77e7e8ee8268)

### 🧐 解决方案

可以考虑两种方案：
1. 只在独立助手内置顶；
2. 全局置顶

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
你点左边的星星图标就是置顶了

[a comment by an automated account is not shown]
